_data/trainings.yml

```diff
@@ -45,3 +45,9 @@
   author: Franziska Sauerwein
   description: For experienced developers who want to improve their design skills in object-oriented languages.
   page: /services/training/oo-design
+
+- id: crafting_swift_apps
+  name: Crafting Swift Mobile Apps
+  author: Ana Nogal
+  description: For experienced mobile developers that want to give the next step with their apps and not only build beautiful apps, but also reliable apps.
```

Comment:
'For experienced mobile developers that want to **give the next step**' does not make sense.

Maybe you mean ...

 'get to the next step' ?

or more simply 

'For experienced mobile developers that want to build reliable and beautiful apps.